src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/entity/SensitiveTaskEventLog.java

```diff
@@ -0,0 +1,170 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.entity;
+
+import com.vladmihalcea.hibernate.type.json.JsonType;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.TypeDef;
+import org.hibernate.annotations.TypeDefs;
+import uk.gov.hmcts.reform.wataskmanagementapi.auth.role.entities.RoleAssignment;
+
+import java.io.Serializable;
+import java.time.OffsetDateTime;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@ToString
+@Getter
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@Entity(name = "sensitive_task_event_logs")
+@TypeDefs(
+    {
+        @TypeDef(
+            name = TaskResource.JSONB,
+            typeClass = JsonType.class
+        )
+    }
+)
+@SuppressWarnings({"PMD.ExcessiveParameterList", "PMD.TooManyFields",
```

Comment:
Do you get all these warnings? I cannot see any UnnecessaryFullyQualifiedName in the code